cpp/src/DataStorm/Instance.cpp

```diff
@@ -56,6 +56,7 @@ Instance::Instance(const Ice::CommunicatorPtr& communicator) : _communicator(com
     if (properties->getPropertyAsIntWithDefault("DataStorm.Node.Multicast.Enabled", 1) > 0)
     {
         properties->setProperty("DataStorm.Node.Adapters.Multicast.Endpoints", "udp -h 239.255.0.1 -p 10000");
```

Comment:
We should create an issue for this port 10000. It should not be hardcoded, and 10000 doesn't look like a sensible default.